packages/solo/src/init-basedir.js

```diff
@@ -82,7 +82,7 @@ export default function initBasedir(
 
   // cosmos-sdk mnemonic and keypair
   if (egresses.includes('cosmos')) {
-    const agchServerDir = path.join(basedir, 'ag-cosmos-helper-statedir');
+    const agchServerDir = path.join(basedir, 'solo-statedir');
```

Comment:
That's a breaking change, from some angles.

In fact, taking away the `ag-cosmos-helper` binary seems like a breaking change.

Reply:
I tried to more aggressively rename in a previous commit that I have since dropped.  Now I've reduced the scope of this PR and marked it breaking.